Makefile

```diff
@@ -11,20 +11,22 @@ test: lint
 lint:
 	pre-commit run -a --verbose golangci-lint
 
+BUILD_FLAGS = -ldflags="-s -w"
```

Comment:
perhaps add a comment with what this does would be good or a link to the article